### PR TITLE
docs: antipattern_intentions

### PR DESCRIPTION
**Pull request by harrisonpim**
adds two intentions
- phrase matching
- synonyms

### CODE REVIEW

File: docs/search_relevance/query_design.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Each query has:
 * **Intentions:** What a person is trying to achieve with their search
 * **Data features:** Parts of the data we think are relevant
 * **Status:** Where we are with developing this query
+
```
Comment (jamesgorrie) [inline docs/search_relevance/query_design.md]:
```suggestion
```

File: docs/search_relevance/query_design.md [hunk 2]
```diff
@@ -28,11 +28,68 @@ Each query has:
...
 
+### Phrase matching
+
+| Status | Query name | Ranking evaluation test |
+| :--- | :--- | :--- |
+| TODO | PhraseMatchQuery | TBD |
+
+#### Intentions
+
+Searching for an exact, ordered set of tokens, in quotation marks
```
Comment (jamesgorrie) [inline docs/search_relevance/query_design.md]: Another PR: have a glossary, add `tokens`.

Comment (jamesgorrie) [reply docs/search_relevance/query_design.md]: https://github.com/wellcomecollection/catalogue/issues/430

**Comment by harrisonpim** [comment]
principles in #431

File: docs/search_relevance/query_design.md (hunk 2 above)
Comment (jamesgorrie) [reply docs/search_relevance/query_design.md]: Or better - `Searching for a phrase`

### glossary 
- phrase: set of ordered tokens
- token: word (<- or whatever)

File: docs/search_relevance/query_design.md [hunk 3]
```diff
@@ -28,11 +28,68 @@ Each query has:
...
+* `data.alternativeTitle`
+* `physicalDescription`
+* `subjects.label`
+* `genres.label`
+* `contributors.label`
+* `description`
+
+#### Expectations
+
+* Searching for a phrase in quotation marks should match works containing that exact set of tokens, in the same order
```
Comment (jamesgorrie) [inline docs/search_relevance/query_design.md]: When using the word phrase do we not already infer order?

Maybe
```suggestion
* Phrases are defined by tokens wrapped in quotation marks and should be matched exactly
```
Or is that too lose?

File: docs/search_relevance/query_design.md [hunk 4]
```diff
@@ -28,11 +28,68 @@ Each query has:
...
+* Works matching individual tokens in the phrase should not be matched by this query
+
+#### Examples
+
+* `The ocean as a health resort : a practical handbook of the sea for the use of tourists and health-seekers`- [https://wellcomecollection.org/works/uxxaqdkg](https://wellcomecollection.org/works/uxxaqdkg) should be the only result returned by this query
```
Comment (jamesgorrie) [inline docs/search_relevance/query_design.md]: Are you sure about it being the only query? Is this then saying if we match a phrase, we don't use other, more loose queries? 

Or is there an expectation that you point that phrase out? Or let the person know that this is highly relevant?

**Comment by jamesgorrie** [comment]
> principles in #431

See https://github.com/wellcomecollection/catalogue/projects/2

File: docs/search_relevance/query_design.md (hunk 4 above)
Comment (jamesgorrie) [reply docs/search_relevance/query_design.md]: > by this query

Makes this obvious.

**Pull request merged**